cli_reference/get_started_cli.adoc

```diff
@@ -24,15 +30,20 @@ Logged into 'https://my-openshift-server.com' as 'myuser'.
 Using project 'project01'.
 Welcome to OpenShift v3! Use 'osc --help' for a list of commands available.
 ----
+====
 
-Three major things happened in the above command run:
+Three major things happened in the above command:
 
-* `osc login` asked for the OpenShift server URL and the user's credentials (_username_ and _password_), established a session in the server, and received a session token.
-* One OpenShift project the user has access to was selected and set as default.
-* All information gathered were saved in a config file located in `~/.config/openshift/.config`.
+- The command prompted for the OpenShift server URL and the login credentials (a user name and password), established a session in the server, and received a session token.
+- One OpenShift project the user has access to was selected and set as default.
+- All information gathered were saved in a config file located in *_~/.config/openshift/.config_*.
 
```

Comment:
I think this would be a great place to use callouts to describe each of these steps, rather than just a bullet list.